### DISCUSSION TITLE
$HOME/QucsWorkspace/user_lib cannot be resolved in Qucs-S-24.3.2-macOSX-x86_64

### DISCUSSION THREAD

**Discussion by bnilsson11** [ra3xdh/qucs_s#1030] [Q&A] [answered]
I am trying to use qucs-s on different (physical) platforms, where I have a user library in $HOME/QucsWorkspace/user_lib.
The component "Lib name of qucs library file" is expressed as an absolute path, which of course is different as seen from the two platforms.
Replacing by $HOME/QucsWorkspace/user_lib/libname.lib is not working.
Should this work, or am I doing it wrong?

**Comment by ra3xdh** [comment]
This will not work. Environment variables are not supported in component properties.

**Comment by bnilsson11** [comment]
So... any solution to make the schematic workplace independent?

**Comment by ra3xdh** [comment, marked as answer]
You have already asked this question. See #922 Put the library in the project directory and specify the relative path. You will have library within project. See also #923 and #567

**Comment by bnilsson11** [comment]
Oops, sorry.
